version/version.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package version
+
+var (
+	// This variable is dependent on what the current release is e.g. if it is v0.10.0 then this variable, outside of releases, will be v0.10.1-dev .
+	Version = "0.4.0-dev" // Change as versions are released
+)
```

Comment:
This should be in `internal` i.e. `internal/version/version.go`

Reply:
Presumably this will also need updated as part of a release. Do any doc updates need to be made to call this out, [here](https://github.com/Kuadrant/dns-operator/blob/main/docs/RELEASE.md) for example? Also the chances of this not getting updated correctly are fairly high so we should probably consider trying to add some automation to ensure it is kept correct. i.e. does it match the version being set at build time. (Don't need to do that in this PR)